SmartCarApplication/app/src/main/java/com/example/smartcarapplication/MainActivity.java

```diff
@@ -144,9 +117,9 @@ public void messageArrived(String topic, MqttMessage message) throws Exception {
                         final byte[] payload = message.getPayload();
                         final int[] colors = new int[IMAGE_WIDTH * IMAGE_HEIGHT];
                         for (int ci = 0; ci < colors.length; ++ci) {
-                            final byte r = payload[3 * ci];
-                            final byte g = payload[3 * ci + 1];
-                            final byte b = payload[3 * ci + 2];
+                            final int r = payload[3 * ci] & 0xFF;
```

Comment:
Can you please explain this to me?

Reply:
this is related to branch 67-implement-smartcar-frontcam, i merged this branch with my branch because i wanted to see how much space camera takes on screen.